xml/admin_ceph_upgrade.xml

```diff
@@ -913,23 +956,23 @@ mds standby for name = mds.456-another-mds
      </para>
     </step>
     <step>
+     <para>
+      On the YaST command line, add the option
+      <option>YAST_ACTIVATE_LUKS=0</option>. This option ensures that no
+      password is asked for for encrypted disks.
+     </para>
+     <warning>
       <para>
-        On the YaST command line, add the option <option>YAST_ACTIVATE_LUKS=0</option>.
-        This option ensures that no password is asked for for encrypted disks.
+       This must not be enabled by default as it would break full-disk
+       encryption on the system disk or part of the system disk. This parameter
+       only works if it is provided the installer. If not provided, you will be
```

Comment:
A preposition is needed here.

provided _by_ the installer? provided _to_ the installer?